bundles/org.openhab.binding.keba/src/main/resources/OH-INF/thing/kecontact.xml

```diff
@@ -21,7 +21,7 @@
 			<channel id="vehicle" typeId="plugvehicle"/>
 			<channel id="locked" typeId="locked"/>
 			<channel id="maxpilotcurrent" typeId="pilotcurrent"/>
-			<channel id="pwmpilotcurrent" typeId="pilotrange"/>
+			<channel id="maxpilotcurrentdutycyle" typeId="pilotcurrentdutycyle"/>
```

Comment:
This change will break backward compatibility. Users, updating to OH 3.1 will need to adapt their config after updating. Are you sure about the change?

Reply:
Updating the config will anyhow be required, as the UoM support was introduced.

The old implementation having `pwmpilotcurrent` and `maxpilotcurrentdutycyle` didn't make sense, as both parameters have been updated with the same value, while `Max curr %` was not used at all:
```
                    case "Max curr": {
                        int state = entry.getValue().getAsInt();
                        maxPresetCurrent = state;
                        updateState(CHANNEL_PILOT_CURRENT, new DecimalType(state));
                        updateState(CHANNEL_PILOT_PWM, new DecimalType(state));
                        break;
                    }
```

The new implementation treats the duty cycle right. It is realized using the right type (`Number:Dimensionless` in %) as well as the name of the parameter.
```
                    case "Max curr": {
                        int state = entry.getValue().getAsInt();
                        maxPresetCurrent = state;
                        State newState = new QuantityType<ElectricCurrent>(state / 1000.0, Units.AMPERE);
                        updateState(CHANNEL_PILOT_CURRENT, newState);
                        break;
                    }
                    case "Max curr %": {
                        int state = entry.getValue().getAsInt();
                        State newState = new QuantityType<Dimensionless>(state / 10.0, Units.PERCENT);
                        updateState(CHANNEL_PILOT_PWM, newState);
                        break;
                    }
```

Reply:
OK. Can you add a short message to https://github.com/openhab/openhab-distro/blob/7dcd280f91fcf9d014c0ca1529bb6ec770a27b4c/distributions/openhab/src/main/resources/bin/update.lst, explaining what has changed concerning backward compatibility?

Reply:
Is this the right file? There is no 3.1.0 nor 3.0.1 section yet. Is this the first breaking change for 3.1.0?

Reply:
I was a bit puzzled, too. But it seems so.

Reply:
https://github.com/openhab/openhab-distro/pull/1263